### ISSUE TITLE
Electoral boundaries need updating

### ISSUE THREAD

**Issue by mackaymackay** [openaustralia/theyvoteforyou#1197]
We received an email this week pointing out the following:

> At least part of post code 3030 (could be the whole postcode, not sure) is no longer part of Lalor federally.
>
> It is now part of Gellibrand, Tim Watts is my federal representative.
>
> You may want to drill down to a street level for post codes that are covered by multiple electorates…

**Comment by jamezpolley** [comment]
See also: openaustralia/openaustralia/issues/632

**Comment by mlandauer** [comment]
This one is super important to get fixed really soon

**Comment by cofiem** [comment]
From a bit of digging, it looks like the way to do this would be to use the 'mesh blocks' from the [ABS Australian Statistical Geography Standard](https://www.abs.gov.au/statistics/standards/australian-statistical-geography-standard-asgs-edition-3/jul2021-jun2026/access-and-downloads/allocation-files).

There are a whole bunch of downloads on that page. I think the way to go would be to get the *Commonwealth Electoral Divisions - 2021* and *Postal Areas - 2021*, then join those two on the 'MB' (mesh block) ids.

The page has other areas, such as indigenous place names and suburb names, which could also be joined using the mesh block ids.

**Comment by cofiem** [comment]
I've implemented a first go at using the ABS Excel file in the [openaustralia-parser repo](https://github.com/openaustralia/openaustralia-parser/pull/137).

**Comment by mlandauer** [comment]
I'm going to take a look at https://github.com/openaustralia/aec_postcode_electorate_data and see if that's easy to get going again.

**Comment by mlandauer** [comment]
Yesterday fixed a problem where none of the repos on morph would get updated from github. Then, it turns out that the AEC site is now blocking morph.io. Fun, fun. Next, need to add paging support to the scraper because some postcode (for example 0820) return multiple pages of results and those are currently being ignored.

**Comment by mlandauer** [comment]
I feel like a bit of a fool because I just realised that by scraping the AEC data now we would get the postcode to electorate mapping for the **upcoming** election. They Vote For You is showing people their **current** MPs so it needs to show the electorates as they were before the election was called. So, if we were to update the postcode to electorate mapping now it would be wrong but what we have now is also a bit wrong.

I think the best thing we can do right now then is to put this issue aside until after the election. Once we've added the members of the new parliament we should update the postcode data. At least most of the work on that has now been done.
